docs/reference/settings/security-settings.asciidoc

```diff
@@ -1986,7 +1988,8 @@ You cannot use this setting and `ssl.truststore.password` at the same time.
 // tag::oidc-ssl-verification-mode-tag[]
 `ssl.verification_mode` {ess-icon}::
 (<<static-cluster-setting,Static>>)
-Controls the verification of certificates.
+Indicates the type of verification when using `oidc` to protect against man
+in the middle attacks and certificate forgery.
```

Comment:
Similar comment here. The reasoning is slightly differently. But overall my suggestion is still that we don't need it here.